WIP: website

ceball opened the pull request:
https://pyviz.github.io/pyviz/ is built on pushing to this PR.

To do:

* [x] Fill in placeholder content, making coherent

Before merging:

* [ ] squash
* [x] update travis file to e.g. build site on tag only

ceball committed:
Initial site (untested).

ceball committed:
Fixed path to built site for gh pages deploy.

ceball committed:
Fix gh pages jekyll thing.

ceball committed:
Add intro to toc.

ceball committed:
Update conf to recent nbsite. Added quick draft logo.

ceball committed:
Fixed syntax error in previous commit.

ceball committed:
Fix links to notebooks. Cut down deployment size.

ceball committed:
Minor changes to build with newer nbsite.

ceball committed:
Include json files in deployed html.

ceball committed:
New nbbuild moves json files.

ceball committed:
Updates for newer nbsite.

ceball committed:
Renamed notebooks to match scheme agreed for nbsite.

ceball committed:
File missed from previous commit.

ceball committed:
Removed system-/user-specific metadata.

ceball committed:
Renamed anacondaviz to pyviz, and added new logo.

ceball committed:
Updated titles.

ceball committed:
Updated some links (still various ones for which I don't know what to do).

ceball committed:
Replaced template content with content from the README.

ceball committed:
Fixed links.

ceball committed:
Minor improvement to text.

jbednar committed:
Updated site text

jbednar committed:
Removed unrelated twitter feed

jbednar commented:
In the commits above, I removed the links to the individual tutorial sections from the main tutorial page, because they had only placeholder text describing them, whereas the 00 Welcome notebook already has full descriptions of each one.  Can we get rid of tutorial/index.rst and simply have it point to tutorial/00_Welcome.html ?  That way users of the website will see the same starting point as users of the notebooks, both of them showing the same content.

Also, right now none of the links to the exercises from 00_Welcome work, because it doesn't look like the exercises are built.  I'm not sure where to add those?

I've added those two tasks to the to-do list above.

jbednar committed:
Fixed minor problems with notebooks

ceball committed:
Stop storing rendered notebooks on appveyor.

We have exceeded the storage capacity, but in any case now we generate a website this is redundant (or will be, when we upload website each time to a location other than the main site).

ceball committed:
Split docs into separate stage. Only uploads docs on tag.

ceball committed:
Updated notebook name.

ceball committed:
Minor changes

[ci skip]
